test/llvm/debugInfo/lldb/basicTypes.good

```diff
@@ -43,8 +43,6 @@ Process XXXX stopped
   }
   myFieldE = NULL
   myFieldF = [1..10] int(64) {
-    dom = 0xXXXX
-    data = 0xXXXX
```

Comment:
This is because you removed the synthetic fields (you mentioned "array gets confused" in the comments)?